tests/sample_data/track_files/rep_data/sen_frig_sensor.dsf

```diff
@@ -1,100 +1,100 @@
-;SENSOR:	700103	033125.000	Frigate	@A	NULL	80	12496	Frigate_Optic some message

```

Comment:
Why are there lots of changes to this file?

Reply:
These two files (`sen_frig_sensor.dsf `and` sen_tracks.rep`) are CRLF. I changed it to LF